web-client/public/index.html

```diff
@@ -27,6 +27,29 @@
         />
         <meta property="og:image" content="/meta.png" />
 
+        <!-- Favicon Images -->
+        <link
+            rel="apple-touch-icon"
+            sizes="180x180"
+            href="/apple-touch-icon.png"
+        />
+        <link
+            rel="icon"
+            type="image/png"
+            sizes="32x32"
+            href="/favicon-32x32.png"
+        />
+        <link
+            rel="icon"
+            type="image/png"
+            sizes="16x16"
+            href="/favicon-16x16.png"
+        />
+        <link rel="manifest" href="/site.webmanifest" />
+        <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#000000" />
```

Comment:
I noticed a white background in a Safari tab vs transparent. Not sure if this is normal safari behavior, or maybe SVG needs a tweak?

<img width="487" alt="Screen Shot 2022-03-15 at 8 07 42 PM" src="https://user-images.githubusercontent.com/139330/158510093-82bb7693-6980-4d6a-a272-b6a2a4e06994.png">

Reply:
Good catch! The svg has a transparent background. I checked it again. The white background is actually only a problem when using darkmode. That's also why I didn't noticed this bug. Our svg is missing contrast: https://stackoverflow.com/questions/59233531/safari-favicon-incorrectly-rendering-with-white-background

The only solution I currently see is using a different favicon for the darkmode in safari, but then it isn't a continuous dao branding anymore...

Reply:
Let's merge since this is an improvement to what we currently have, and Safari darkmode is not a show stopper. Thanks @noahliechti for your work on this!